packages/service/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2018. All Rights Reserved.
+Node module: @loopback/service
```

Comment:
Let's use a more descriptive name than `@loopback/service` please. For example, I would expect `@loopback/service` to provide functionality for _building_ services.

Few proposals:
 - `@loopback/service-client`
 - `@loopback/service-integration`
 - `@loopback/service-proxy`
 - `@loopback/service-repository`